templates/ts/notification-restify/README.md.tpl

```diff
@@ -12,32 +12,49 @@ The app template is built using the TeamsFx SDK, which provides a simple set of
 > To run the notification bot template in your local dev machine, you will need:
 >
 > - [Node.js](https://nodejs.org/), supported versions: 16, 18
+{{^enableTestToolByDefault}}
```

Comment:
The M365 account is still a prerequisite if uses want to debug in Teams, so I don't think it can be removed from the README.md directly.